sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -116,6 +194,15 @@ export default defineComponent({
       getRemoveBypassLabel,
       getRemoveBypassColor,
       ApplicationRestrictionManagementHeaders,
+      Role,
+      openBypassRestrictionModal,
+      bypassRestrictionModal,
+      openRemoveBypassModal,
+      removeBypassModal,
+      reloadRestrictionBypassedHistory,
```

Comment:
No need to return `reloadRestrictionBypassedHistory` since it is not consumed if the template.